### DISCUSSION TITLE
当前是否推荐大家使用2.2.0版本

### DISCUSSION THREAD

**Discussion by wangsz05** [halo-dev/halo#3223] [General]
HI all 
我想咨询下管理员，当前是否推荐大家使用2.2.0版本，还是说继续使用1.X版本。

**Comment by ruibaby** [comment]
新建的网站无疑用 Halo 2 是最好的，如果迁移可以根据 https://docs.halo.run/getting-started/migrate-from-1.x 检查一下是否满足要求。

1.x 将在不久后停止维护。
